### PR TITLE
AC-675: Disabled brightback for dev environment

### PR DESCRIPTION
**Pull request by PatrickSison**
### What Changed
 - Added enabled flag to brightback config
- Added enabled:false to dev brightback config
- Brightback does not prep or send when downgrading from paid to free plan

### How To Test
 - Using a paid account, downgrade from paid to free on all environments

### To Do
- [ ] Double check on staging and prod
- [ ] Address any feedback

### CODE REVIEW

**Comment by PatrickSison** [comment]
Not sure if the default config should be disabled w/ test configs enabling vs disabling w/ prod enabling

**Comment by rajumsys** [comment]
> Not sure if the default config should be disabled w/ test configs enabling vs disabling w/ prod enabling
Does it work (or supposed to) in env other than prod? if no, then let's disable by default and enable only in prod. 

Otherwise LGTM

**Comment by PatrickSison** [comment]
Rebased

File: src/components/brightback/Brightback.js [hunk 1]
```diff
@@ -34,6 +36,7 @@ Brightback.propTypes = BrightbackPropTypes;
 const mapStateToProps = (state, props) => ({
   valid: state.brightback.valid,
   url: state.brightback.url,
-  data: selectBrightbackData(state, props)
+  data: selectBrightbackData(state, props),
+  enabled: configFlag('brightback.enabled')(state)
```
Comment (rajumsys) [inline src/components/brightback/Brightback.js]: does it need `state` as arg? i don't see how it's being used.

Comment (PatrickSison) [reply src/components/brightback/Brightback.js]: It doesn't need it. Removed it

**Pull request merged**